docs/generate-time-series/javascript.rst

```diff
@@ -0,0 +1,232 @@
+    ...   database: 'doc',
+    ...   port: 5432,
+    ... })
+    > await client.connect()
+
+The client can also be initialised with a single connection string, if prefered::
+
+    > const client = new Client({
+    ...   connectionString: 'postgresql://crate@localhost:5432/doc'
+    ... })
+    > await client.connect()
+
+Then, finally, `create a table`_ suitable for writing load averages::
+
+    > var query = `
+    ...     CREATE TABLE load (
+    ...         timestamp TIMESTAMP GENERATED ALWAYS AS CURRENT_TIMESTAMP, 
+    ...         avg_1m REAL, 
+    ...         avg_5m REAL, 
+    ...         avg_15m REAL)`
+    > await client.query(query)
+
+
+In the `CrateDB Admin UI`_, you should see the new table when you navigate to
+the *Tables* screen using the left-hand navigation menu:
+
+.. image:: ../_assets/img/generate-time-series/table.png
+
+
+Record your system load
+=======================
+
+With the table in place, you can start recording load averages.
+
+The following command calls `os.loadavg`_ and uses the result triple as `input
+values`_ for the `INSERT`_ query::
+
+    > await client.query("INSERT INTO load (avg_1m, avg_5m, avg_15m) VALUES (?, ?, ?)", os.loadavg())
+
+Press the up arrow on your keyboard and hit *Enter* to run the same command a
+few more times.
+
+When you're done, you can `SELECT`_ that data back out of CrateDB, like so::
+
+    > client.query('SELECT * FROM load ORDER BY timestamp DESC', (err, res) => {
+    ...   if (err) {
+    ...       console.log(err.stack)
+    ...     } else {
+    ...       console.log(res.rows)
+    ...     }
+    ... })
+    [ { timestamp: 2020-02-13T14:23:23.566Z,
+        avg_1m: 0.83251953,
+        avg_5m: 0.54785156,
+        avg_15m: 0.2265625 },
+      { timestamp: 2020-02-13T14:23:22.968Z,
+        avg_1m: 0.90527344,
+       avg_5m: 0.5571289,
+        avg_15m: 0.22802734 },
+      { timestamp: 2020-02-13T14:23:21.144Z,
+        avg_1m: 0.90527344,
+        avg_5m: 0.5571289,
+        avg_15m: 0.22802734 } ]
+
+Here we have recorded three sets of load averages with a corresponding
+timestamp. We can now close the connection::
+
+    > await client.end()
+
+
+Automate it
+===========
+
+Now we have the basics figured out, let's automate the data collection.
+
+Our approach will change a little bit here. We were previously using a `Client`_ to
+connect to and insert data into CrateDB. However, Clients are ephemeral, and once closed
+they need to be recreated. However, creating a new Client requires a handshake with CrateDB, and
+this overhead cost can be prohibative if we are creating new clients rapidly.
+
+Instead, we can use a `Connection Pool`_ to manage our connections. Create a new file called
+``monitor-load.js``, like this:
+
+.. code-block:: javascript
+
+    const os = require('os')
+    const { Pool } = require('pg')
+    const pool = new Pool({
+        user: 'crate',
+        host: 'localhost',
+        database: 'doc',
+        port: 5432,
```

Comment:
we should make sure this matches whichever connection method you choose as the one to recommend for this tutorial (assuming you agree with my comment above)